safir/src/safir/uws/_app.py

```diff
@@ -82,13 +71,7 @@ class UWSApplication:
     def __init__(self, config: UWSConfig) -> None:
```

Comment:
Really for lines 56-63 (stupid GitHub...): change the docstring for `UWSApplication` to reflect that it calls wobbly rather than works directly with a database.